anyway/parsers/cbs/importmail_cbs.py

```diff
@@ -8,14 +8,15 @@
 from datetime import datetime
 
 from anyway.utilities import time_delta
+from anyway import secrets
 
 mail_dir = "cbs/data"
 
 
 def main(detach_dir, username=None, password=None, email_search_start_date=""):
     try:
-        username = username or os.environ.get("MAILUSER")
-        password = password or os.environ.get("MAILPASS")
+        username = username or secrets.get("MAILUSER")
+        password = password or secrets.get("MAILPASS")
```

Comment:
Note that `username` is not a secret, though it is convenient here to treat it that way.
Note also that `password` is not _really_ expected to be None; see the comment for `get()` below.

(I'm not suggesting any particular action here.)